www/.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
+  root: true,
+  extends: [
+    '../.eslintrc.js',
+  ],
+  rules: {
+    'import/no-extraneous-dependencies': 0,
+  },
+  settings: {
+    'import/resolver': {
+      webpack: {
+        config: {
+          resolve: {
+            alias: {
```

Comment:
Arguably the nicest change here - this will finally allow use to alias imports like `import { Card } from ~paragon-react` without eslint complaining (which we previously suppressed with `eslint-disable-next-line` in like 20 files)